src/composables/accounts.ts

```diff
@@ -253,27 +254,27 @@ export function useAccounts() {
   /**
    * Setting/Resetting the password key logs the user in/out.
    */
-  function setEncryptionData(newEncryptionData: IEncryptionData | null) {
-    encryptionData.value = newEncryptionData;
+  function setEncryptionKey(newEncryptionKey: CryptoKey | null) {
+    encryptionKey.value = newEncryptionKey;
     if (IS_EXTENSION) {
-      if (newEncryptionData) {
-        startSession(newEncryptionData, secureLoginTimeout.value);
+      if (newEncryptionKey) {
+        startSession(newEncryptionKey, secureLoginTimeout.value);
       } else {
         endSession();
       }
     }
   }
 
-  async function getEncryptionData() {
-    await watchUntilTruthy(encryptionData);
-    return encryptionData.value;
+  async function getEncryptionKey() {
+    await watchUntilTruthy(encryptionKey);
+    return encryptionKey.value!;
```

Comment:
With `encryptionData` this would result in a weird type instead of the correct type of `encryptionData`
Now with `encryptionKey` its not happening so I'll follow your suggestion